### ISSUE TITLE
Error: expected `&Client`, found `&ClientInitGuard`

### ISSUE THREAD

**Issue by arashi-dev** [timfish/sentry-tauri#8]
I want to add sentry plugin to my code but I am getting this error:
```
78 |     let _guard = sentry_tauri::minidump::init(&client);
   |                  ---------------------------- ^^^^^^^ expected `&Client`, found `&ClientInitGuard`
   |                  |
   |                  arguments to this function are incorrect
```

here is my code:
```rs
let client = sentry_tauri::sentry::init((
        "__YOUR_DSN__",
        sentry_tauri::sentry::ClientOptions {
            release: sentry_tauri::sentry::release_name!(),
            ..Default::default()
        },
    ));
    
let _guard = sentry_tauri::minidump::init(&client);
```

and here is my `Cargo.toml`:
```toml
[dependencies]
tauri = { version = "1.5.4", features = ["..."] }
sentry-tauri = "0.2"
```

How can I fix it?

**Comment by timfish** [comment]
I just published v0.3.0 which should fix this.

**Comment by tance77** [comment]
@timfish 

this seems to be happening again with the new `sentry 0.37`

```sh
   --> src/main.rs:124:61
    |
124 |         .plugin(tauri_plugin_sentry::init_with_no_injection(&client))
    |                 ------------------------------------------- ^^^^^^^ expected `&Client`, found `&ClientInitGuard`
    |                 |
    |                 arguments to this function are incorrect
    |
    = note: expected reference `&tauri_plugin_sentry::sentry::Client`
               found reference `&sentry::ClientInitGuard`
note: function defined here
```
